.github/workflows/ci.yaml

```diff
@@ -29,12 +32,12 @@ jobs:
         run: echo "::set-output name=image::${{ env.IMAGE }}"
 
   linters:
+    if: ${{ !github.event.act }}
     name: ament-${{ matrix.linter }}
     needs: prepare-actions-environment
     runs-on: ubuntu-latest
     container:
       image: ${{ needs.prepare-actions-environment.outputs.image }}
-      options: -u root
```

Comment:
just curious, why was this removed?

Reply:
Oh this was me messing around to see if `rosdep update` was causing the issue. `rosdep` does not like being run as root, but that was not the issue nor do I think this line of code touches `rosdep`; readded. Nice catch.